openmetadata-ui/src/main/resources/ui/cypress/e2e/Features/ActivityFeed.spec.js

```diff
@@ -0,0 +1,192 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { interceptURL, verifyResponseStatusCode } from '../../common/common';
+
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+
+const reactOnFeed = (feedSelector, reaction) => {
+  cy.get(feedSelector).within(() => {
+    cy.get('.feed-actions').invoke('show');
+    cy.get('.feed-actions').within(() => {
+      cy.get('[data-testid="add-reactions"]').click();
+    });
+  });
+
+  cy.get(
+    `#reaction-popover [data-testid="reaction-button"][title="${reaction}"]`
+  ).click();
+};
+
+describe('Recently viwed data assets', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('Feed widget should be visible', () => {
+    cy.get('[data-testid="activity-feed-widget"]').as('feedWidget');
+    cy.get('@feedWidget').should('be.visible');
+    cy.get('@feedWidget').contains('All');
+    cy.get('@feedWidget').contains('@Mentions');
+    cy.get('@feedWidget').contains('Tasks');
+    cy.get('@feedWidget').contains('0');
```

Comment:
.contains will select that element, to assert the element please use .should("contains",KEY_WORD)